Unable to build nativeTest under GraalVM Native Image in parallel

- Unable to build nativeTest under GraalVM Native Image in parallel.
```shell
git clone git@github.com:linghengqian/hive-server2-jdbc-driver.git
cd ./hive-server2-jdbc-driver/
sdk use java 22.0.2-graalce
./mvnw -T 1.5C -PnativeTestInCustom clean test
```
- The log is as follows.
```shell
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for io.github.linghengqian:hive-parent 1.0.0:
[INFO] 
[INFO] io.github.linghengqian:hive-parent ................. SUCCESS [  0.080 s]
[INFO] hive-server2-jdbc-driver-thin ...................... SUCCESS [01:58 min]
[INFO] hive-server2-jdbc-driver-uber ...................... FAILURE [ 32.732 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:58 min (Wall Clock)
[INFO] Finished at: 2024-08-15T15:27:00+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.graalvm.buildtools:native-maven-plugin:0.10.2:test (test-native) on project hive-server2-jdbc-driver-uber: Execution test-native of goal org.graalvm.buildtools:native-maven-plugin:0.10.2:test failed.: FileSystemNotFoundException -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :hive-server2-jdbc-driver-uber

```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -62,4 +62,4 @@ jobs:
           github-token: ${{ secrets.GITHUB_TOKEN }}
           cache: 'maven'
       - run: |
-          ./mvnw -PnativeTestInCustom clean test
+          ./mvnw -T 1.5C -PnativeTestInCustom clean test

--- a/doc/CONTRIBUTING.md
+++ b/doc/CONTRIBUTING.md
@@ -28,7 +28,7 @@ sdk use java 22.0.2-graalce
 git clone git@github.com:linghengqian/hive-server2-jdbc-driver.git
 cd ./hive-server2-jdbc-driver/
 sdk use java 22.0.2-graalce
-./mvnw -PnativeTestInCustom clean test
+./mvnw -T 1.5C -PnativeTestInCustom clean test
 ```
 
 ### Fixes LICENSE issue

